.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,80 @@
+name: Bug Report
+description: File a bug report
```

i must keep my quirkiness
```suggestion
description: Something's wrong...
```